miner/worker.go

```diff
@@ -1133,30 +1133,11 @@ func (w *worker) commitTransactions(env *environment, txs *types.TransactionsByP
 		if len(mvReadMapList) > 0 {
 			tempDeps := make([][]uint64, len(mvReadMapList))
 
-			// adding for txIdx = 0
-			tempDeps[0] = []uint64{uint64(0)}
-			tempDeps[0] = append(tempDeps[0], 1)
-
 			for j := range deps[0] {
 				tempDeps[0] = append(tempDeps[0], uint64(j))
 			}
 
 			for i := 1; i <= len(mvReadMapList)-1; i++ {
-				tempDeps[i] = []uint64{uint64(i)}
-
-				reads := mvReadMapList[i-1]
-
-				_, ok1 := reads[blockstm.NewSubpathKey(env.coinbase, state.BalancePath)]
-				_, ok2 := reads[blockstm.NewSubpathKey(common.HexToAddress(w.chainConfig.Bor.CalculateBurntContract(env.header.Number.Uint64())), state.BalancePath)]
-
-				if ok1 || ok2 {
-					// 0 -> delay is not allowed
-					tempDeps[i] = append(tempDeps[i], 0)
```

Comment:
I think we should still keep this check. When delay is not allowed, we simply set dependency to empty, so validators will directly run serial execution.

Reply:
Yes, that will be nice.